en-US/main.lang

```diff
@@ -657,3 +657,43 @@ Thanks for choosing Firefox!
 # Used in combination with other strings to create HTML page titles, e.g. in What's new pages
 ;Mozilla Firefox Web Browser
 Mozilla Firefox Web Browser
+
+
+# A social link in the footer
+;Twitter
```

Comment:
Drop all social network names: Twitter, Facebook, Instagram, YouTube. We don't want them localized or transliterated (they were also in the current footer)